server/tweetSearch.js

```diff
@@ -324,12 +324,22 @@ module.exports = function(client, fs, speakerFile) {
     }
 
     function addSpeaker(name) {
+        tweetUpdates.push({
```

Comment:
I had no particular reason for doing it that way so I've changed it, let me know if it's what you had in mind.